modin/core/dataframe/algebra/groupby.py

```diff
@@ -107,16 +123,16 @@ def register_implementation(cls, map_func, reduce_func):
     @classmethod
     def map(
         cls,
-        df,
-        map_func,
-        axis,
-        groupby_kwargs,
-        agg_args,
-        agg_kwargs,
-        other=None,
+        df: pandas.DataFrame,
+        map_func: Callable[..., pandas.DataFrame],
+        axis: int,
+        groupby_kwargs: dict,
+        agg_args: list,
+        agg_kwargs: dict,
+        other: Optional[pandas.DataFrame] = None,
         by=None,
```

Comment:
Why is this untouched?

Reply:
Some arguments are very complex in terms of the types they accept. I leave them untouched, because for now I am trying to improve code navigation with the least effort, so as not to spend a lot of time on it and be able to do it as I work on another task (but in the same place in the code).